app/web_ui/src/routes/(app)/evals/[project_id]/[task_id]/[eval_id]/eval_configs/+page.svelte

```diff
@@ -50,7 +50,20 @@
     | "pearson"
     | "kendalltau"
 
-  let score_type: ScoreType = "kendalltau"
+  let score_type: ScoreType | null = null
+
+  // Set default score_type to "mse" when eval rating type is pass_fail or pass_fail_critical, otherwise use "kendalltau"
```

Comment:
I'd comment on the "why" here more than the what. The code says what it does, but it's a bit of a weird thing to do, so a comment on why we are doing this is probably helpful.